packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js

```diff
@@ -0,0 +1,133 @@
+/* eslint no-underscore-dangle: 0  */
+
+import React from 'react';
+import {Select, SelectOption, RadioGroup, Radio} from '@momentum-ui/react';
+import * as AdaptiveCard from 'adaptivecards';
+import classnames from 'classnames';
+import {v4 as UUID} from 'uuid';
+import {renderCardElement, CHOOSE_ONE, CHOOSE} from '@ciscospark/react-component-utils';
+
+/**
+ * Replaces Choice Set(radio button and checkboxes) of adaptive Card with MD(Momentum Design) Select and Radio Group
+ * @returns {HTMLElement}
+ */
+class MDChoiceSetInput extends AdaptiveCard.ChoiceSetInput {
+  constructor(addChildNode) {
+    super();
+    this.addChildNode = addChildNode;
+  }
+
+  /**
+   * returns the assigned value to internalRender() method
+   * @returns {string}
+   */
+  get value() {
+    return this._value;
+  }
+
+  /**
+   * assigns the value when the data is sent in the JSON request
+   * @param {string} newValue
+   */
+  set value(newValue) {
+    this._value = newValue;
+  }
+
+  /**
+   * Renders the momentum design `<Select />` and `<RadioGroup />` components
+   * @returns {HTMLElement}
+   */
+  internalRender() {
+    this._selectElement = document.createElement('div');
```

Comment:
I don't understand why we are not creating the `divs` in a react way.

Reply:
This isn't react per se. We're actually rendering a React component inside a DOM node. Please refer: https://reactjs.org/docs/integrating-with-other-libraries.html#integrating-with-other-view-libraries